tests/integration.rs

```diff
@@ -3469,6 +3470,45 @@ mod parallel {
         handle_child_output(r, &output);
     }
 
+    #[test]
+    #[cfg(not(feature = "acpi"))]
+    #[cfg(target_arch = "aarch64")]
+    fn test_pmu_on() {
```

Comment:
You're talking about 2 tests but added only one here.

Reply:
Thank you @sboeuf for this, this commit message reflects the one we had in internal code review, and we decided that the "negative" test is not really necessary. We will rephrase the commit message.